src/Decomposition/CartDecomposition.hpp

```diff
@@ -392,17 +392,17 @@ class CartDecomposition: public ie_loc_ghost<dim, T,layout_base, Memory>,
 		dec_optimizer<dim, Graph_CSR<nm_v<dim>, nm_e>> d_o(dist.getGraph(), gr_dist.getSize());
 
 		// Ghost
-		Ghost<dim,long int> ghe;
+		Ghost<dim,long int> ghostExtended;
 
 		// Set the ghost
 		for (size_t i = 0 ; i < dim ; i++)
 		{
-			ghe.setLow(i,static_cast<long int>(ghost.getLow(i)/spacing[i]) - 1);
-			ghe.setHigh(i,static_cast<long int>(ghost.getHigh(i)/spacing[i]) + 1);
+			ghostExtended.setLow(i,static_cast<long int>(ghost.getLow(i)/spacing[i]) - 1);
```

Comment:
Since `ghe` is `Ghost` which is technically `Box`  aand the dimensions of the original `ghost` are used here, I assume it is `ghostExtended` and not `ghostExtension`.